testsuite/classlibrary/TestServer_dumpTree.sc

```diff
@@ -0,0 +1,64 @@
+TestServer_dumpTree : UnitTest {
+
+    var pipe;
+
+    setUp {
+    }
+
+    tearDown {
+        if (pipe.notNil) {
+            pipe.close;
+        };
+    }
+
+    getOutput { |program, expectedOutput|
+        var actualOutput = List[],
+            addr = NetAddr("127.0.0.1", 57110),
+            line;
+        pipe = Pipe.new(program ++ ServerOptions.new.asOptionsString, "r");
```

Comment:
Sorry, I just realized something.
57110 should probably be a variable - use it for NetAddr AND as an argument for ServerOptions - `ServerOptions().asOptionsString(port)` - just so there's no ambiguity there.

I guess `ServerOptions().bindAddress_(ip)` for could be also specified explicitly, I guess. So:
```supercollider
ip = "127.0.0.1";
port = 57110;
addr = NetAddr(ip, port);

pipe = Pipe.new(program ++ ServerOptions().bindAddress_(ip).asOptionsString(port), "r")
```